Sources/SwiftInspectorCommands/InitializerCommand.swift

```diff
@@ -71,9 +71,9 @@ final class InitializerCommand: ParsableCommand {
 
   private func outputString(from statement: InitializerStatement) -> String {
     if typeOnly {
-      return statement.parameters.map { $0.typeName }.joined(separator: " ")
+      return statement.parameters.map { $0.typeNames.joined(separator: " ") }.joined(separator: " ")
```

Comment:
I'm a bit confused by how you parse the output now. It seems like it would be hard to parse `foo,MyProtocol YourProtocol anotherParam:String`.

Should it be `foo,MyProtocol,YourProtocol anotherParam:String` instead?

Reply:
In the case where you exclude parameter names, how do you know which types belong to switch initializer parameter? How do I parse `MyProtocol YouProtocol String`?

Reply:
@bachand this is for the type only case. So this would output:

`MyProtocol YourProtocol String` in your use case.

[Below](https://github.com/fdiaz/SwiftInspector/pull/21/files#diff-f8fee815716808d2efcdd15e9654898aR76) is the update for multiple types when outputting the argument name. It'd look like:

`foo,MyProtocol,YourProtocol anotherParam,String`

Reply:
@fdiaz I'm curious why you chose to output `MyProtocol YourProtocol String` in the type only case vs. `MyProtocol,YourProtocol String`. In any case, I'm fine deferring this until we address https://github.com/fdiaz/SwiftInspector/issues/23

Reply:
I think its easier to parse when we do that, and if we're saying we only care about the types I don't see a value in adding context for each argument.

What I'm guessing this can be used to is to know what types a certain class takes in case there are types we want to forbid or warn about.

Since we have the `--disable-type-only` flag, if we do care about the parameters and how they're grouped together we can always get that information from there instead.

Reply:
> What I'm guessing this can be used to is to know what types a certain class takes in case there are types we want to forbid or warn about.

Makes sense!